docs/section5/Rule5-4.md

```diff
@@ -24,20 +24,20 @@
 
 - For each building segment in the Baseline model: ```for building_segment_baseline in B_RMR.building.building_segments:```  
```

Comment:
@JasonGlazer and @mcneillj the applicability check identified above (conditioned spaces , walls etc are determined correctly) are identified as 'Rule Dependencies' in the TCD spreadsheet, which is why I believe @xingcx has added this here.
However, this is not really an 'applicability check' the way we're defining this right? if a check, this should be an RMR test OR we should use the surface tilt and H/C capacities to determine the SCC and surface type. should we remove these applicability checks?

Reply:
@supriyagoel I think I mixed up the files for the PR somehow. Please ignore all changes for Rule5-4 here. RDS/XC/Rule5-4 should cover all the latest changes. I moved this comment to RDS/XC/rule5-4.